NOTES.md

```diff
@@ -1,3 +1,105 @@
+# Release 2.1.0-beta (Das Boot)
```

Comment:
The movie title is "Forbidden Planet" (https://en.wikipedia.org/wiki/Forbidden_Planet)